ls1intum/Artemis
# [Bugfix] - Fix student statistics reachable scores calculation

### Checklist

### Motivation and Context


- Reachable score was calculated incorrectly
- Definition of reachable points: Reachable points are the points which the student could theoretically already have achieved. This includes points of exercises, where the assessment is already been finished. (-> after the assessmentDueDate)

### Description

- The participation is only relevant for the points achieved by the student and should not be used for the calculation of the reachable score or the max score

### Steps for Testing


1. Log in to Artemis
2. Navigate to the statistics page in the course student view
3. Check that the reachable points are correct (see definition of reachable points above)
